stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -49,6 +49,9 @@ public class ChannelListViewModel(
     public val chatDomain: ChatDomain,
     initialSort: QuerySort<Channel>,
     private val initialFilters: FilterObject,
+    channelLimit: Int = ChannelViewModelFactory.DEFAULT_CHANNEL_LIMIT,
```

Comment:
I think `ChannelListViewModel` should be completely unaware of `ChannelViewModelFactory` class. What do you think about leaving the constants in the ViewModel?

Reply:
Agreed! Will push to the VM and then call from the factory!